playbooks/roles/filebeat/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+- name: Ensure {{ filebeat_app_dir }} exists
+  file: path={{ filebeat_app_dir }} state=directory owner=root group=root mode=0744
+
+- name: ensure we have the specified filebeat release
+  get_url: url={{ filebeat_url }} dest=/tmp/{{ filebeat_file }}
+
+- name: Install filebeat
+  shell: dpkg --skip-same-version -i /tmp/{{ filebeat_file }}
+  register: dpkg_result
+
+- name: copy config
+  template:
+    src: filebeat.yml.j2
+    dest: "/etc/filebeat/filebeat.yml"
+    mode: 0744
+  notify: restart filebeat
+
+- name: create symlink to {{ filebeat_app_dir }} for config
+  file: src="/etc/filebeat/filebeat.yml" dest="{{ filebeat_app_dir }}/filebeat.yml" state=link
```

Comment:
You know, even though this is convention elsewhere, it's not a particularly good one.
Any documentation refers to the default paths in `/etc/`, and we'll probably end up installing this on machines that otherwise have no connection to `/edx/`.

So, I'm thinking we can drop the symlink here, and just always refer to it as `/etc/filebeat/filebeat.yml`